packages/genesys-spark-components/src/components/stable/gux-icon/example.html

```diff
@@ -1,244 +1,12 @@
-<h1>gux-icon</h1>
```

Comment:
This file will not be commit when I merge this PR it is changed now to help review the change